practice-app/backend/tests/covid19.spec.js

```diff
@@ -0,0 +1,27 @@
+const chai = require('chai'),
+    mocha = require('mocha'),
+    app = require('../app').App,
+    Constants = require('../constants'),
```

Comment:
This line gives error when I run the npm test command. I think you forgot to delete constants part from this file

Reply:
You're right. I'm going to remove it.